src/pages/_app.tsx

```diff
@@ -26,10 +35,15 @@ class MyApp extends App<MyAppProps & ReduxWrapperAppProps<RootState>> {
 
   render() {
     const { Component, pageProps, colorMode, store } = this.props;
+
+    // Workaround for https://github.com/zeit/next.js/issues/8592
+    const { err } = this.props;
```

Comment:
`err` should be taken our from above, no need to have two instances of destructuring this.props.